store/src/java/com/zimbra/cs/account/ldap/LdapProvisioning.java

```diff
@@ -10570,6 +10575,11 @@ private void addDynamicGroupMembers(LdapDynamicGroup group, String[] members)
             String memberName = member.toLowerCase();
             memberName = IDNUtil.toAsciiEmail(memberName);
 
+            Account account = getAccountByName(memberName);
```

Comment:
`getAccountByName()` calls `get(AccountBy.name, name)` to fetch the account, so getting account null on line **10579** is invalid account and on line **10583** getting account null could is being checked for **group/group unit address** isn't it contradicting each other ?